.travis.yml

```diff
@@ -14,6 +14,7 @@ install:
   - mkdir -p $GOPATH/src/github.com/btcsuite/
   - "[ -d $GOPATH/src/github.com/btcsuite/btcd ] || git clone https://github.com/btcsuite/btcd.git $GOPATH/src/github.com/btcsuite/btcd"
   - pushd $GOPATH/src/github.com/btcsuite/btcd
+  - git checkout .
```

Comment:
This should be addressed if we pin to the latest btcd version.

Reply:
Partly addressed in #187. Should be fully addressed by adding the tag to the `btcd` go.mod

Reply:
Should be done in its own PR when we update the dependency

Reply:
Why can't we just update the dependency now and get rid of this commit?

Reply:
Will update the dep after btcd's `v0.20.0` tag, then we can both fetch that and pin against that in our `go.mod`.